uspv/init.go

```diff
@@ -72,8 +69,42 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		if err != nil {
 			return err
 		}
+		err := s.Handshake(nil)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+func (s *SPVCon) ConnectToSeedAdrs( seedAdrs []string) error {
```

Comment:
This can be reworked into a DialNode() function; takes in a [single / slice of] addresses and tries to connect, assigning that connection to s.con.

It can just return an error, and doesn't need to call any sub functions; after it returns (without error) there will be an s.con that's waiting for a handshake.